src/core/server/server.api.md

```diff
@@ -546,18 +553,11 @@ export type RequestHandler<P extends ObjectType, Q extends ObjectType, B extends
 // @public
 export type ResponseError = string | Error | {
     message: string | Error;
-    meta?: ResponseErrorMeta;
+    attributes?: ResponseErrorAttributes;
 };
 
 // @public
-export interface ResponseErrorMeta {
-    // (undocumented)
-    data?: Record<string, any>;
-    // (undocumented)
-    docLink?: string;

```

Comment:
I really liked the idea of typing API best practices like this. It's not critical now, but hopefully we can re-introduce this at some point when we can make breaking changes with HAPI.